server/src/test/java/io/crate/fdw/JdbcForeignDataWrapperTest.java

```diff
@@ -74,5 +74,31 @@ public void test_cannot_access_localhost_as_regular_user() throws Exception {
         assertThatThrownBy(() -> fdw.getIterator(arthur, server, foreignTable, txnCtx, List.of(nameRef), Literal.BOOLEAN_TRUE))
             .hasMessage("Only a super user can connect to localhost unless `fdw.allow_local` is set to true");
     }
+
+    @Test
+    public void test_can_access_remote_as_regular_user() throws Exception {
+        Functions functions = new Functions(Map.of());
+        Role arthur = RolesHelper.userOf("arthur");
+        Roles roles = () -> List.of(arthur);
+        NodeContext nodeCtx = new NodeContext(functions, roles);
+        var fdw = new JdbcForeignDataWrapper(Settings.EMPTY, new InputFactory(nodeCtx));
+        Settings options = Settings.builder()
+            .put("url", "jdbc:postgresql://123.123.123.123:5432/postgres")
```

Comment:
Maybe use an IP from https://www.rfc-editor.org/rfc/rfc5737#section-3
I think 123.123.123.123 is a real/valid IP.